travis-ci/test-data/custom_targets.json

```diff
@@ -0,0 +1,28 @@
+{
+    "IMAGINARYBOARD": {
+        "inherits": [
+            "MCU_STM32L475xG"
+        ],
+        "components_add": [
+            "BlueNRG_MS",
+            "QSPIF"
+        ],
+        "extra_labels_add": [
+            "CORDIO",
+            "MX25R6435F"
+        ],
+        "supported_form_factors": [
+            "ARDUINO"
+        ],
+        "detect_code": [
+            "1234"
+        ],
+        "device_has_add": [
+            "QSPI"
+        ],
+        "features": [
+            "BLE"
+        ],
+        "device_name": "STM32L475VG"
```

Comment:
Should we set this to something imaginary?

Reply:
No, we shouldn't. This is an MCU attribute, not a board one. This name must match a name given in a CMSIS-Pack for the MCU, so that debugging and flashing tools that understand CMSIS-Pack can find and work with this board.